example/10_conv2d_bwd_data/conv2d_bwd_data_xdl.cpp

```diff
@@ -180,6 +180,10 @@ int main(int argc, char* argv[])
     out_device_buf.ToDevice(out_n_k_ho_wo.mData.data());
     wei_device_buf.ToDevice(wei_k_c_y_x.mData.data());
 
+    // reset input to zero
```

Comment:
Please use a GPU kernel to initialize buffer to 0.

Reply:
Use function hipMemset to  set all memory  zero or write a single kernel to set a tensor  zero?

Reply:
please use this, in following up PR
https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/21457e7aea94c9fb92ea1cf79ecafea2197f3011/example/15_gemm_reduce/gemm_reduce_xdl_fp16.cpp#L209